### PR TITLE
Make data directory and TCP ports for demo cluster configurable on the fly

### LINKED ISSUE #441: Master port for demo cluster is not configurable
The TCP port used by "make cluster" is hardcoded to 15432, which makes it impossible to use when two builds are tested at the same time.

This port should be configurable by a commandline option or an environment variable.

### PR DESCRIPTION
**Pull request by andreasscherbaum**
…e fly

The data directory and the TCP ports can be changed when "make cluster" is
executed to build the demo cluster:

DATADIRS=/tmp/gpdb-cluster MASTER_PORT=15432 PORT_BASE=25432 make cluster

This fixes #441 and #442

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,11 @@ make cluster
 source gpdemo-env.sh
 ```
 
+The directory and the TCP ports for the demo cluster can be changed on the fly:
+
+DATADIRS=/tmp/gpdb-cluster MASTER_PORT=15432 PORT_BASE=25432 make cluster
```
Comment (danielgustafsson) [inline README.md]: This row should be wrapped in code block markers (\`\`\` foo \`\`\`) for consistency.

**Comment by danielgustafsson** [comment]
With the documentation altered, :+1:

**Comment by andreasscherbaum** [comment]
Updated the commit. Thanks for the feedback.

**Comment by andreasscherbaum** [comment]
Please do not merge this yet. I will look into the regression tests as well.

**Comment by andreasscherbaum** [comment]
@danielgustafsson Can you have one last look, please? I made the regression tests configurable as well. That ideally should cover everything now.

**Comment by danielgustafsson** [comment]
:+1: on the updated version.

**Comment by ascherbaum-pivotal** [comment]
Pushed.

**Pull request closed without merging**